source/proof_address/lib/proof_address.rb

```diff
@@ -9,7 +9,7 @@
 module IdentityIdpFunctions
   class ProofAddress
     def self.handle(event:, context:, &callback_block)
```

Comment:
this affects the calling code in the IDP so update that when I bump this in the IDP